### PR TITLE
[WIP2] Fix MSVS for current master

### PR DESCRIPTION
**Pull request by mwojcikowski**

### CODE REVIEW

**Comment by mwojcikowski** [comment]
Now it builds fine on Appveyor on both MSVS 2008 and 2015

**Comment by mwojcikowski** [comment]
Here you have Appveyor channel which builds conda packages: https://ci.appveyor.com/project/mwojcikowski/openbabel-packages/history

One problem i have is that  files are not installed properly (see [bld.bat](https://github.com/mwojcikowski/openbabel-packages/blob/master/conda/bld.bat))

File: src/data.cpp [hunk 1]
```diff
@@ -1056,6 +1056,10 @@ namespace OpenBabel
     // Check return value from OpenDatafile
     // Suggestion from Zhiguo Liu
     string fn_open = OpenDatafile(ifs, _filename, _envvar);
+    
+    // Check _subdir directory
```
Comment (mwojcikowski) [inline src/data.cpp]: This does not work, I dont really have a clue what to try next.

Comment (ghutchis) [inline src/data.cpp]: I dunno. I'm inclined to merge this for 2.4.0 - which has been postponed *far* too long and then work hard on a 2.4.1. Sound OK?

Comment (mwojcikowski) [inline src/data.cpp]: Other than that line this PR is solid I'd say. It fixes few things for windows. We just have to work out the "data dir". I second merging it as is and figuring the relatively small bug with Windows "data dir" later on.

**Pull request merged**